variables.tf

```
@@ -48,13 +48,13 @@ variable "dataset_labels" {
   type        = map(string)
 }
 
-variable "table_labels" {
-  description = "Key value pairs in a map for table labels"
-  type        = map(string)
-}
-
 variable "tables" {
   description = "A list of maps that includes both table_id and schema in each element, the table(s) will be created on the single dataset"
```

The descriptions of the `tables` variables should be updated to reflect the new type.

```suggestion
  description = "A list of objects which include table_id, schema, and labels."
```